iroha_data_model/src/lib.rs

```diff
@@ -357,10 +361,23 @@ pub mod world {
         /// Iroha `Triggers` registered on the peer.
         pub triggers: Vec<Instruction>,
         /// Iroha parameters.
-        pub parameters: Vec<Parameter>,
+        pub parameters: RwLock<Vec<Parameter>>,
```

Comment:
Parameters can be a lockfree set

Reply:
I guess parameters semantically need temporal ordering. Hashset is not ordered in any way.